engine/install/sles.md

```diff
@@ -0,0 +1,269 @@
+---
+description: Instructions for installing Docker Engine on SLES
+keywords: requirements, apt, installation, centos, rpm, sles, install, uninstall, upgrade, update
+redirect_from:
+- /ee/docker-ee/sles/
+- /engine/installation/sles/
+- /engine/installation/linux/sles/
+- /engine/installation/linux/docker-ce/sles/
+- /engine/installation/linux/docker-ee/sles/
+- /engine/installation/linux/docker-ee/sles/
+- /engine/installation/linux/sles/
+- /engine/installation/sles/
+- /install/linux/sles/
+- /install/linux/docker-ce/sles/
+- /install/linux/docker-ee/sles/
+- /installation/sles/
+title: Install Docker Engine on SLES
+toc_max: 4
+---
+
+To get started with Docker Engine on SLES, make sure you
+[meet the prerequisites](#prerequisites), then
+[install Docker](#installation-methods).
+
+## Prerequisites
+
```

Comment:
Given that this distro is only available on s390x currently, perhaps it's good to highlight that a bit more, e.g.:

```suggestion
## Prerequisites

> **Note**
>
> We currently only provide packages for SLES on s390x (IBM Z). Other architectures
> are not yet supported for SLES.

```